{
  "operation_results": [
    {
      "result": {
        "auto_dismissed_at": null,
        "created_at": "2022-06-15T07:43:03Z",
        "dependency": {
          "manifest_path": "path/to/requirements.txt",
          "package": {
            "ecosystem": "pip",
            "name": "django"
          },
          "scope": "runtime"
        },
        "dismissed_at": "2022-08-23T14:29:47Z",
        "dismissed_by": {
          "avatar_url": "https://github.com/images/error/octocat_happy.gif",
          "email": null,
          "events_url": "https://api.github.com/users/octocat/events{/privacy}",
          "followers_url": "https://api.github.com/users/octocat/followers",
          "following_url": "https://api.github.com/users/octocat/following{/other_user}",
          "gists_url": "https://api.github.com/users/octocat/gists{/gist_id}",
          "gravatar_id": "",
          "html_url": "https://github.com/octocat",
          "id": 1,
          "login": "octocat",
          "name": null,
          "node_id": "MDQ6VXNlcjE=",
          "organizations_url": "https://api.github.com/users/octocat/orgs",
          "received_events_url": "https://api.github.com/users/octocat/received_events",
          "repos_url": "https://api.github.com/users/octocat/repos",
          "site_admin": false,
          "starred_at": null,
          "starred_url": "https://api.github.com/users/octocat/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/octocat/subscriptions",
          "type": "User",
          "url": "https://api.github.com/users/octocat",
          "user_view_type": null
        },
        "dismissed_comment": "This alert is accurate but we use a sanitizer.",
        "dismissed_reason": "tolerable_risk",
        "fixed_at": null,
        "html_url": "https://github.com/octocat/hello-world/security/dependabot/2",
        "number": 2,
        "security_advisory": {
          "cve_id": "CVE-2018-6188",
          "cvss": {
            "score": 7.5,
            "vector_string": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N"
          },
          "cvss_severities": {
            "cvss_v3": {
              "score": 7.5,
              "vector_string": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N"
            },
            "cvss_v4": {
              "score": 8.7,
              "vector_string": "CVSS:4.0/AV:N/AC:L/AT:N/PR:N/UI:N/VC:H/VI:N/VA:N/SC:N/SI:N/SA:N"
            }
          },
          "cwes": [
            {
              "cwe_id": "CWE-200",
              "name": "Exposure of Sensitive Information to an Unauthorized Actor"
            }
          ],
          "description": "django.contrib.auth.forms.AuthenticationForm in Django 2.0 before 2.0.2, and 1.11.8 and 1.11.9, allows remote attackers to obtain potentially sensitive information by leveraging data exposure from the confirm_login_allowed() method, as demonstrated by discovering whether a user account is inactive.",
          "ghsa_id": "GHSA-rf4j-j272-fj86",
          "identifiers": [
            {
              "type": "GHSA",
              "value": "GHSA-rf4j-j272-fj86"
            },
            {
              "type": "CVE",
              "value": "CVE-2018-6188"
            }
          ],
          "published_at": "2018-10-03T21:13:54Z",
          "references": [
            {
              "url": "https://nvd.nist.gov/vuln/detail/CVE-2018-6188"
            },
            {
              "url": "https://github.com/advisories/GHSA-rf4j-j272-fj86"
            },
            {
              "url": "https://usn.ubuntu.com/3559-1/"
            },
            {
              "url": "https://www.djangoproject.com/weblog/2018/feb/01/security-releases/"
            },
            {
              "url": "http://www.securitytracker.com/id/1040422"
            }
          ],
          "severity": "high",
          "summary": "Django allows remote attackers to obtain potentially sensitive information by leveraging data exposure from the confirm_login_allowed() method, as demonstrated by discovering whether a user account is inactive",
          "updated_at": "2022-04-26T18:35:37Z",
          "vulnerabilities": [
            {
              "first_patched_version": {
                "identifier": "2.0.2"
              },
              "package": {
                "ecosystem": "pip",
                "name": "django"
              },
              "severity": "high",
              "vulnerable_version_range": "\u003e= 2.0.0, \u003c 2.0.2"
            },
            {
              "first_patched_version": {
                "identifier": "1.11.10"
              },
              "package": {
                "ecosystem": "pip",
                "name": "django"
              },
              "severity": "high",
              "vulnerable_version_range": "\u003e= 1.11.8, \u003c 1.11.10"
            }
          ],
          "withdrawn_at": null
        },
        "security_vulnerability": {
          "first_patched_version": {
            "identifier": "2.0.2"
          },
          "package": {
            "ecosystem": "pip",
            "name": "django"
          },
          "severity": "high",
          "vulnerable_version_range": "\u003e= 2.0.0, \u003c 2.0.2"
        },
        "state": "dismissed",
        "updated_at": "2022-08-23T14:29:47Z",
        "url": "https://api.github.com/repos/octocat/hello-world/dependabot/alerts/2"
      },
      "type": "procedure"
    }
  ]
}